### DISCUSSION TITLE
macOS関係の情報交換

### DISCUSSION THREAD

**Discussion by CSReviser** [CSReviser/CaptureStream2#24] [General]
Macユーザーの皆さんのmacOS版に関連した情報提供など情報交換の場として作りました。

**Comment by CSReviser** [comment]
ターミナルアプリを開いて

xattr -dr com.apple.quarantine 

最後に半角スペースが入った状態でダウンロードした.dmgファイル　または　.appファイルをマウスでターミナルアプリにドロップするとパスを含めたファイル名が入力されるのでEnterを押す。

xattr -cr 
codesign -s - 

も同様に実施する。
これでセキュリティ警告を回避出来ないでしょうか？

**Reply by CSReviser** [reply]
[「開発元が不明なMacアプリを開く」](https://support.apple.com/ja-jp/guide/mac-help/mh40616/mac)

[「アプリがMac App Storeから入手されていない場合」](https://support.apple.com/ja-jp/guide/mac-help/mh40620/mac)

上記がApple Macユーザガイドの公式手順になるようです。バージョンを選択して使用環境に合わせた手順で対応して下さい。
macOS版のインストール手順を修正します。

**Comment by CSReviser** [comment]
**macOS版で同一バージョンでも複数ファイルがあるので、どのファイルを使えば良いかわからないという方のための情報です。**

**（１）CaptureStream2-MacOS-20yymmdd.dmg**
　　**<例>CaptureStream2-MacOS-20241007.dmg**
* **通常はこのファイルを使ってください。**
* **Apple Silicon（Mシリーズ） MacではRosetta2が必要です。**
　　　


**（２）CaptureStream2-MacOS-arm-20yymmdd.dmg**
　　**<例>CaptureStream2-MacOS-arm-20241007.dmg**
* **Apple Silicon(Mシリーズ)を想定していますが、MシリーズのMacでも（１）が使えるはずです。**

　　　


**（３）CaptureStream2-MacOS-non-ffmpeg-20yymmdd.dmg**
　　**<例>CaptureStream2-MacOS-non-ffmpeg-20241007.dmg**
* **既に動作するffmpegをインストール済みの場合を想定していますが、この場合でも（１）が使えるはずです。**
　
　　　

**（４）CaptureStream2-MacOS-qt5-Intel-20yymmdd.dmg**
　　**<例>CaptureStream2-MacOS-qt5-Intel-20241007.dmg**
* **OSX10.13など古いOSで使う場合を想定しています。特殊なケース以外は使いません。**

**Comment by nakayamajp** [comment]
Tahoeで従来バージョンが正常動作しましたのでご報告いたします（Mac mini M1）。

**Comment by DaddyJapan** [comment]
以下の2つのファイルの場所が特定できました。参考です。
~/Library/Preferences/CaptureStream.ini
~/Applications/CaptureStream.app/Contents/MacOS/stylesheet-mac.qss

**Reply by CSReviser** [reply]
はい、macOS版はOS側の仕様の制限によって語学講座CS2の仕様を他のOSと変えています。
[MacOSユーザの方へのページの＜MacOS版の仕様相違点＞](https://csreviser.github.io/CaptureStream2/macos#macos%E7%89%88%E3%81%AE%E4%BB%95%E6%A7%98%E7%9B%B8%E9%81%95%E7%82%B9)に記載しています。
style sheetは保存フォルダに置くとそちらを優先する仕様です。2026年度版では他のOSもmacOS版と同じ仕様(プログラム上共通のコード)にしています。

**Comment by 2gj7xvc7bx-star** [comment]
2026年度更新　Qt 6.11 build test
Release 20260329 - 2026/03/29版リリース
の動作を確認しました。Macbook Pro M4

**Comment by DaddyJapan** [comment]
ありがとうございました。style sheetはテストし、動作を確認してしました。引き続き、新しい版が出ましたら、テストしたいと思います。動作には直接影響しませんが、以下のカスタマイズの文字が少し見えづらいと思われました。背景色を白から別の色に変えると見やすくなると思われました。よろしくお願い致します。On Mar 30, 2026, at 5:40, CSReviser ***@***.***> wrote:﻿
はい、macOS版はOS側の仕様の制限によって語学講座CS2の仕様を他のOSと変えています。
MacOSユーザの方へのページの＜MacOS版の仕様相違点＞に記載しています。
style sheetは保存フォルダに置くとそちらを優先する仕様です。2026年度版では他のOSもmacOS版と同じ仕様(プログラム上共通のコード)にしています。

**Reply by DaddyJapan** [reply]
2026/04/05版をテストしてみました。画面は以下のようになり、見やすくなりました。ありがとうございます。
[Doc1.docx](https://github.com/user-attachments/files/26487284/Doc1.docx)

**Comment by DaddyJapan** [comment]
2026/04/05版で、拡張子の選択をしようとしたときに、選択肢が見えにくい状態でした。
また、fontを変えたら、サムネ付の文字が欠けてしまいました。
[Doc1.docx](https://github.com/user-attachments/files/26488361/Doc1.docx)

**Reply by CSReviser** [reply]
ダークモードでしょうか？
通常モードでも出る現象でしょうか？

**Comment by DaddyJapan** [comment]
通常モードでは問題ありません。Darkモードだけですか、実用的にはこのままでも使えます。
[Doc1.docx](https://github.com/user-attachments/files/26491419/Doc1.docx)

**Reply by CSReviser** [reply]
2026/04/06版でダークモード向けに調整しました。まだ課題がありますが、ダークモードでもそれなりの表示になると思います。これ以上は今後の課題としたいと思います。

**Comment by DaddyJapan** [comment]
保存フォルダーのstylesheet-mac.qssにあるフォントサイズの変更で、サムネ付もきれいに出力されました。
[Doc1.docx](https://github.com/user-attachments/files/26491646/Doc1.docx)

**Comment by DaddyJapan** [comment]
2026/04/06版の結果です。問題は無いようです。fontは変えています。Darkモードです。
[Doc1.docx](https://github.com/user-attachments/files/26507791/Doc1.docx)

**Comment by DaddyJapan** [comment]
CaptureStream2-MacOS-20260407.dmgに更新し、問題ないことをMacbookで確認しました。
Upgradeの表示も出たので、更新がわかるようになり、便利になりました。
ありがとうございました。
[Doc1.docx](https://github.com/user-attachments/files/26588467/Doc1.docx)

**Comment by acchi3** [comment]
いつもありがとうございます。CaptureStream2-MacOS-20260407.dmgについて、iMac（intel）Sequoia 15.7.4で動作するのを確認しました。
